index.bs

```diff
@@ -7003,6 +7047,17 @@ five forms are allowed.
         identifier "=" identifier
 </pre>
 
+<pre class="grammar" id="prod-ExtendedAttributeString">
+    ExtendedAttributeString :
+        identifier "=" string
+</pre>
+
+<pre class="grammar" id="prod-ExtendedAttributeNumber">
```

Comment:
This apparently breaks the LL(1) of the grammar so needs investigating.

Reply:
Splitting them ExtendedAttributeNumber into ExtendedAttributeDecimal and ExtendedAttributeInteger seems to fix this. I also wonder do we actually need to handle integer or can we just use decimal and do "2.0" when we want "2"?

Reply:
I'd definitely prefer to use integers for `unsigned long long` etc. attributes. It's unnecessarily confusing otherwise.